judgments/views/advanced_search.py

```diff
@@ -47,14 +47,22 @@ def advanced_search(request):
     * Given anything except an HTTP GET request raise an error
     """
     # We should only be handling GET requests here since we aren't changing anything on the server
-    if request.method == "GET":
+    if not request.method == "GET":
+        # Raise an error if the user has tried any non GET HTTP requests.
+        raise Http404("GET requests only")
+    else:
```

Comment:
Since we're raising at line 52, we can drop everything else back an indentation.